bedrock/base/templates/macros.html

```diff
@@ -144,7 +144,7 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
   </section>
 {%- endmacro %}
 
-{# Docs: https://bedrock.readthedocs.io/en/latest/firefox-accounts.html#linking-to-accountsfirefoxcom #}
+{# Docs: https://bedrock.readthedocs.io/en/latest/firefox-accounts-2019.html#linking-to-accountsfirefoxcom #}
```

Comment:
The changes in this file are not related to this issue, there's no need to update these links.